### PR TITLE
Bluetooth cubes forbiddance clarification

### PR DESCRIPTION
**Pull request by Ivan-Brigidano**
See #665 and #783 
I think this way is the most logical way to clarify that this cubes aren't allowed

### CODE REVIEW

**Comment by Ivan-Brigidano** [comment]
Maybe it’s better to incloude it into 3k)?

**Comment by hanwu85** [comment]
I don't think it's necessary to included it in 3k, they are talking about different aspect of things.
Regarding travis error, there is a white space at line 118.

File: wca-guidelines.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ To be more informative, each Guideline is classified using one of the following
 - 2h+) [CLARIFICATION] Competitors may be barefoot for 3x3x3 With Feet.
 - 2i+) [ADDITION] Although the competitor may pick up a stopwatch to view the current time (when they are not blindfolded), they must not start, stop, pause, or otherwise interact with the timekeeping of the stopwatch.
 - 2i++) [ADDITION] The organization team may provide the competitor an unofficial stopwatch for viewing the elapsed time (started together with the main stopwatch), in which case the competitor is not permitted to touch the official stopwatch.
+- 2i+++) [REMINDER] Bluetooth puzzles are considered electric devices.
```
Comment (lgarron) [inline wca-guidelines.md]: Small tweak:

```suggestion
- 2i+++) [REMINDER] Bluetooth puzzles are considered electronic devices.
```

File: wca-regulations.md [hunk 2]
```diff
@@ -115,6 +115,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3a1) Competitors must be ready to submit their puzzles when they are called (see [Regulation 2u](regulations:regulation:2u)).
     - 3a2) Puzzles must be fully operational, such that normal scrambling is possible.
     - 3a3) Polyhedral puzzles must use a color scheme with one unique color per face in the solved state. Each puzzle variation must have moves, states, and solutions functionally identical to the original puzzle.
+    - 3a4) Puzzles must not have bluetooth and/or wi-fi functions (see [Regulation 2i](regulations:regulation:2i)).
```
Comment (lgarron) [inline wca-regulations.md]: I'd do something like this:

```suggestion
    - 3a4) Puzzles must not have electronic components (e.g. Bluetooth or Wi-Fi capabilities, motors, sensors, lights). See [Regulation 2i](regulations:regulation:2i).
```

**Review by lgarron — CHANGES_REQUESTED** [review]

**Comment by lgarron** [comment]
Thanks for submitting this!

**Review by lgarron — APPROVED** [review]

**Pull request merged**